@xen-orchestra/xapi/src/vm.js

```diff
@@ -485,9 +486,14 @@ module.exports = class Vm {
       ref = await this.callAsync($cancelToken, 'VM.snapshot', vmRef, nameLabel).then(extractOpaqueRef)
     } while (false)
 
-    // Don't set `is_a_template = false` like done in xo-server, it does not
-    // appear necessary and can trigger license issues, see
-    // https://bugs.xenserver.org/browse/XSO-766
+    // required in order to ensure that the full backup will not be imported as a template
+    await pCatch.call(
+      this.setField('VM', ref, 'is_a_template', false),
+
+      // see https://bugs.xenserver.org/browse/XSO-766
+      { code: 'LICENSE_RESTRICTION' },
+      Function.prototype
```

Comment:
Avoid using `Function.prototype` directly, IMHO, it's not easily understandable.

It's better to define it on top of the file as `noop` and then use it.